src/components/manage/Blocks/Maps/View.jsx

```diff
@@ -38,7 +38,9 @@ const View = ({ data, intl }) => (
       })}
     >
       <iframe
-        title={intl.formatMessage(messages.EmbededGoogleMaps)}
+        title={`${intl.formatMessage(messages.EmbededGoogleMaps)} ${
+          data.title
+        }`}
```

Comment:
@iRohitSingh I think we should simply show data.title as it was entered in the block schema, and not add "Embedded Google Maps of " as a prefix. Maybe the editor wants to provide a more detailed description of the map that does not fit grammatically with the prefix. Maybe some site customizes the view of this block to display maps from another source instead of Google.

(If we do keep the prefix, the spelling needs to be fixed: `Embedded` has 3 Ds.)